hw6_task_2.py

```diff
@@ -0,0 +1,44 @@
+import math
+
+if __name__ == "__main__":
+    def find_square_data(side: int):
```

Comment:
declaration of function should be outside if->main. This block is only for executing